File: 01-Introduction/01-E.py
def main() -> None:
    # Read from stdin.
    input = [int(x) for x in open(0)]
                
    # Execute function on every line after the first.
    # Described behaviour is quivalent to quadratic growth.
    # Pretty print.
    print(*[(x+1)**2 for x in input[1:]], sep = '\n')


if __name__ == '__main__':
    main()

File: 01-Introduction/01-E.cpp
#include <iostream>

int main() {
    // Read integers from stdin.
    int cases;
    unsigned long long int n; // Need 10^18 digits.
    std::cin >> cases;

    // Loop over cases.
    for (int i = 0; i < cases; i++) {
        std::cin >> n;
        // Described behaviour is equivalent to quadratic growth.
        std::cout << (n+1)*(n+1) << std::endl;
    }
    return 0;
}